lib/components/note-preview/index.tsx

```diff
@@ -29,12 +31,46 @@ type Props = OwnProps & StateProps & DispatchProps;
 export const NotePreview: FunctionComponent<Props> = ({
   editNote,
   fontSize,
+  isDialogOpen,
+  isNoteInfoOpen,
   note,
   noteId,
   searchQuery,
 }) => {
   const previewNode = useRef<HTMLDivElement>();
 
+  useEffect(() => {
+    const copyRenderedNote = (event: ClipboardEvent) => {
+      console.log(isNoteInfoOpen);
+      // TODO this isn't working - these are false when I log them, but in console they are true
+      // e.g. state.ui.showNoteInfo in console will print "true"
```

Comment:
the reason for this oddity is that we're enclosing the copies of the variables inside this closure when we create it. that is, when grabbing them from the outer props they have been bound to `isNoteInfoOpen` and `isDialogOpen` with the values they had when `useEffect()` ran.

with `this` referring to a component's backing instance we can still access the props at the time of invocation because we follow the reference through the objects to the values of `props` (vs reading a copied value).

one way around this is to add those values as dependencies to the `useEffect()` and have it recreate the closure when they change.

```ts
useEffect(() => {
	…
}, [isDialogOpen, isNoteInfoOpen])
```

That should recreate the effect whenever those props change and update the values bound inside of it.